app/models/setting.rb

```diff
@@ -170,7 +172,9 @@ def self.create!(opts)
 
   def self.create_existing(s, opts)
     bypass_readonly(s) do
-      to_update = Hash[opts.select { |k,v| [:default, :description].include? k }]
+      attrs = [:default, :description]
+      attrs << :full_name if Setting.column_names.include?('full_name')
+      to_update = Hash[opts.select { |k,v| attrs.include? k }]
       to_update.merge!(:value => SETTINGS[opts[:name].to_sym]) if SETTINGS.key?(opts[:name].to_sym)
       s.update_attributes(to_update)
       s.update_column :category, opts[:category] if s.category != opts[:category]
```

Comment:
I think you need a similar line, as the `s.update_attributes` isn't actually setting full_name in the database for me.  If I debug it, I see there's an error on the model:

    @messages={:full_name=>["»is not allowed to change«"]